flocker/volume/httpapi.py

```diff
@@ -10,27 +13,41 @@
 from klein import Klein
 
 from ..restapi import structured
+from .. import __version__
+
+
+SCHEMA_BASE = FilePath(__file__).parent().child(b'schema')
+SCHEMAS = {
+    b'/v1/types.json': yaml.safe_load(
+        SCHEMA_BASE.child(b'types.yml').getContent()),
+    b'/v1/endpoints.json': yaml.safe_load(
+        SCHEMA_BASE.child(b'endpoints.yml').getContent()),
+    }
 
 
-class DatasetAPIUser(object):
+class DatasetAPIUserV1(object):
     """
     A user accessing the API.
     """
     app = Klein()
 
-    @app.route("/noop")
-    @structured({}, {})
-    def noop(self):
+    @app.route("/version")
```

Comment:
I'll leave that for the followup api documentation ticket https://clusterhq.atlassian.net/browse/FLOC-101